pkg/urbit/noun/nock.c

```diff
@@ -1695,6 +1872,11 @@ _n_hilt_fore(u3_noun hin, u3_noun bus, u3_noun* out)
       *out = u3_nul;
     } break;
 
+    case c3__xray : {
+      _xray(0, fol);
+      *out = u3_nul;
+    } break;
```

Comment:
This version of the hint shouldn't have any caption at all. I think that `_slog_bytecode()` just just produce `u3i_string()` (a trivial tank), and the different hint implementations should `u3t_slog()` or `u3t_slog_cap()` as appropriate.

Reply:
rewrote so that _slog_bytecode just slogs, no slog with caption, thus the two call sites will render as expected and either render one caption or none at all.

Reply:
I think we should go one step further, as I described.

- `_cn_slog_bytecode` should be renamed to `_cn_etch_bytecode()`, and simply return `u3i_string(str_c)`
- the case block here should call `u3t_slog(u3nc(0, _cn_etch_bytecode()))`
- the case block below should call `u3t_slog_cap()`